csc/cmd/controller_list_snapshots.go

```diff
@@ -75,6 +79,18 @@ func init() {
 		"",
 		"The starting token used to retrieve paged data")
 
+	listSnapshotsCmd.Flags().StringVar(
+		&listSnapshots.sourceVolumeId,
+		"source-volume-id",
+		"",
+		"The starting token used to retrieve source-volume id")
```

Comment:
Please update the description of the flag to be accurate. This is not to specify a starting token. From the spec:

>   // Identity information for the source volume. This field is OPTIONAL.
  // It can be used to list snapshots by volume.

So I would instead say something like "ID of volume to list snapshots for"

Reply:
Done